README.md

```diff
@@ -23,7 +23,7 @@ Add your answers to this markdown file.
 
 In order to verify the Computer Science Researchers claim that this proprietary sorting algorithm can sort arbitrary elements in O(n) time based on comparisons of two elements, there are two main steps I would take in approaching the problem
 1. Emperical Testing
-   - In order to verify the Computer Science Researchers claim that this proprietary sorting algorithm can sort arbitrary elements in O(n) time based on comparisons of two elements, there are two main steps I would take in approaching the problem
+   - Due to having access to the algorithm through a black-box implementation, I would test it with a range of input sizes. Then I would analyze the time it takes to complete each sort. For this there are specifically three main things that will take place: 
        - Generating Test Cases:
```

Comment:
What if the empirical testing shows that it's possible to have O(n) complexity? I mean, there are cases where it could be O(n). Please explain when those exceptions could occur.

Reply:
Okay I'm really sorry although I'm a little confused. I thought I included this! In the plotting times sections I discuss if the complexity is O(n) it should show a linear relationship. And then I also include an expected results section also including this information. To my understanding that's what you're asking me to explain, but if this is not correct please let me know!

Reply:
I mean, when the complexity is O(n) in empirical testing, it means some exceptional structure in the, for example, array to be sorted, such as the array is already sorted, exactly the thing we saw in the Insertion sort. Or maybe the sorting algorithm follows non-comparison methods like Bucket Sort. Please provide this explanation in the Readme file, as well. Thank you.